djcelery/models.py

```diff
@@ -182,6 +184,8 @@ class PeriodicTasks(models.Model):
     ident = models.SmallIntegerField(default=1, primary_key=True, unique=True)
     last_update = models.DateTimeField(null=False)
 
+    no_changes = False
```

Comment:
What is the purpose of this line here? Maybe it should be moved to `IntervalSchedule`?